code/game/gamemodes/clock_cult/clock_items/judicial_visor.dm

```diff
@@ -169,7 +169,7 @@
 		var/datum/status_effect/belligerent/B = C.apply_status_effect(STATUS_EFFECT_BELLIGERENT)
 		if(!QDELETED(B))
 			B.duration = world.time + 30
-			C.Knockdown(5) //knocks down for half a second if affected
+			C.Knockdown(90) //knocks down for 9 seconds if affected
```

Comment:
this is actually very silly since this is applied *on-click*

Reply:
like, there's a 29 decisecond delay plus a slow, so maybe make it stun for like 26 so someone you clicked on can't move out before it blows up, then make the hit stun bigger?